### PR TITLE
fix connection pagination (without fix to return correct order for last/before)

### PR DESCRIPTION
**Pull request by jedwards1211**
@mickhansen  Here is my fix with the old, incorrect edges order, as you requested

### CODE REVIEW

**Comment by jedwards1211** [comment]
@mickhansen @janmeier are you guys waiting for anything on this?

The test failures were as usual due to the postgres race condition.

**Comment by mickhansen** [comment]
Hey @jedwards1211, sorry - have just been busy.

File: src/relay.js [hunk 1]
```diff
@@ -107,6 +107,127 @@ export function nodeType(connectionType) {
...
+
+
+/**
+ * Creates a cursor given a node returned from the Database
+ * @param  {Object}   node            sequelize model instance
+ * @param  {String[]} orderAttributes  the attributes pertaining in ordering
+ * @return {String}                   The Base64 encoded cursor string
+ */
+function toCursor(node, info) {
+  return base64(JSON.stringify(info.orderAttributes.map(attr => node.get(attr))));
```
Comment (mickhansen) [inline src/relay.js]: this makes it a migration-breaker,
any way we can make it so old cursors would still work?

File: src/relay.js [hunk 2]
```diff
@@ -107,6 +107,127 @@ export function nodeType(connectionType) {
...
+  const {sequelize, QueryGenerator} = model;
+  const attributesStr = attributes.map(attribute => QueryGenerator.quoteIdentifier(attribute)).join(', ');
+  const escapeOptions = {context: 'SELECT'};
+  const valuesStr = attributes.map((attribute, i) =>
+    QueryGenerator.escape(values[i], model.attributes[attribute], escapeOptions)
+  ).join(', ');
+  return sequelize.literal(`(${attributesStr}) ${inequality} (${valuesStr})`);
+}
+
+function getWindow({model, cursor, order, inclusive}) {
```
Comment (mickhansen) [inline src/relay.js]: can you provide an example of what SQL this would generate?

File: src/relay.js (hunk 1 above)
Comment (jedwards1211) [reply src/relay.js]: I'm confused what you mean, cursors are just opaque strings, and there's no way an old-style cursor would somehow pop up in the new code

File: src/relay.js (hunk 2 above)
Comment (jedwards1211) [reply src/relay.js]: Basically `(name, id) < ('foo', 5)` for instance

File: src/relay.js (hunk 1 above)
Comment (jedwards1211) [reply src/relay.js]: I mean unless users do something weird like persisting cursors until they upgrade, which we should discourage people from doing

File: src/relay.js (hunk 2 above)
Comment (jedwards1211) [reply src/relay.js]: Instead of `name < 'foo' OR (name = 'foo' AND id < 5)`

**Pull request closed without merging**